src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,26 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_AGE = 18;
+    private static final int MIN_PASSWORD_LENGTH = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user.getLogin() == null || user.getAge() == null || user.getPassword() == null) {
+            throw new RuntimeException("Value should not be null");
+        }
+        if (user.getAge() < MIN_AGE) {
+            throw new RuntimeException("User's minimum age must be at least 18");
+        }
+        if (user.getPassword().length() < MIN_PASSWORD_LENGTH) {
+            throw new RuntimeException("Password's minimum length "
+                    + "should be at least 6 characters");
```

Comment:
What if min age or min password length change in future? You will change only value of constant and all messages will be changed after it automatically. 
```suggestion
                    + "should be at least " + MIN_PASSWORD_LENGTH + " characters");
```